.pre-commit-config.yaml

```diff
@@ -51,3 +51,9 @@ repos:
     hooks:
       - id: black
         exclude: scripts/templates/reference/|Testing/Tools/cxxtest
+
+  - repo: https://github.com/jhaigh0/mantid-pre-commit
```

Comment:
Shouldn't the hook be in https://github.com/mantidproject/pre-commit-hooks.git ?